packages/report-server/src/reportBuilder/output/functions/rawDataExport/rawDataExport.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2022 Beyond Essential Systems Pty Ltd
+ */
+
+import { yup } from '@tupaia/utils';
+
+import { ReportServerAggregator } from '../../../../aggregator';
+import { Row } from '../../../types';
+import { OutputContext } from '../../types';
+import { RawDataExportBuilder } from './rawDataExportBuilder';
+import { RawDataExport, RawDataExportContext } from './types';
+
+const contextValidator = yup.object().shape({
+  dataGroups: yup.array().of(yup.string().required()).required().min(1),
+});
+
+const rawDataExport = async (
+  rows: Row[],
+  params: unknown,
+  outputContext: RawDataExportContext,
+  aggregator: ReportServerAggregator,
+): Promise<RawDataExport> => {
+  return new RawDataExportBuilder(rows, params, outputContext, aggregator).build();
+};
+
+const buildParams = (params: unknown): unknown => {
+  return params;
+};
```

Comment:
Still wanna keep buildParams for consistent format over the places.

Reply:
Good call 👍